db/migrate/20210128143531_add_primary_contact_email_to_school.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPrimaryContactEmailToSchool < ActiveRecord::Migration[6.1]
+  def change
+    add_column :schools, :primary_contact_email, :string
+  end
```

Comment:
https://github.com/DFE-Digital/early-careers-framework/pull/71 adds that too... Arghhhhhh

Whatever. This likely should go in first.